src/sbt-test/sbt-typescript-plugin/module-resolution/app/assets/javascripts/es6-shim.d.ts

```diff
@@ -0,0 +1,670 @@
+// Compiled using typings@0.6.10
```

Comment:
This file is needed only for compilation and should be there on plugin's user side too. Ignore it while reviewing.